config/sql/se/Ebooks.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `Ebooks` (
+  `EbookId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `AbsoluteUrl` varchar(255) NOT NULL,
+  `Title` varchar(255) NOT NULL,
+  `Created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `Updated` timestamp NOT NULL,
```

Comment:
Yep, good call. I'll add that in a followup PR because there are a few renames I'd like to make. The `Ebook` class already has these two members:

* `Created`
* `Updated`

that I would like to rename to 

* `DatePublished`
* `DateModified`

or simply

* `Published`
* `Modified`

to reflect where they come from and what they represent. Then the DB schema will have:

* `Created`
* `Updated`
* `Published`
* `Modified`

to reflect both when the DB record was created/updated and when the ebook was published/modified.